map_model/src/lane.rs

```diff
@@ -212,7 +211,7 @@ impl Lane {
         }
     }
 
-    pub fn get_turn_restrictions(&self, road: &Road) -> Option<BTreeSet<TurnType>> {
+    pub fn get_turn_restrictions<'slf, 'road>(&'slf self, road: &'slf Road) -> Option<impl Iterator<Item=TurnType> + 'slf> {
```

Comment:
Same here -- and `'road` is unused. I'm kind of surprised rustc doesn't warn about this